lib/materialist/workers/event.rb

```diff
@@ -16,8 +16,9 @@ def perform(event)
 
         report_latency(topic, timestamp) if timestamp
         report_stats(topic, action, :success)
-      rescue
+      rescue Exception => exception
         report_stats(topic, action, :failure)
+        notice_error(exception, event)
         raise
```

Comment:
I'm unsure whether this will cause double logging in new relic. _going to test..._

Reply:
if you `raise` again, then yes. I think we should simply not `raise` again and perhaps have a default `notice_error` lambda that does, to keep the existing behaviour?

Reply:
This will work, but once concern I have with this approach is that it will be easy to forget to `raise` when you are providing a `handle_error` which then will totally (and possibly seemlessly) turn off any retry. 

What do you think?